components/constellation/constellation.rs

```diff
@@ -277,11 +276,16 @@ pub struct Constellation<STF, SWF> {
     /// to facilitate installing pipeline namespaces in threads
     /// via a per-process installer.
     namespace_receiver: RoutedReceiver<PipelineNamespaceRequest>,
-    namespace_ipc_sender: GenericSender<PipelineNamespaceRequest>,
+    pub(crate) namespace_ipc_sender: GenericSender<PipelineNamespaceRequest>,
+
+    /// A [`Vec`] of all [`EventLoop`]s that have been created for this [`Constellation`].
+    /// This will be cleaned up periodically. This stores weak references so that [`EventLoop`]s
```

Comment:
Where does this cleanup happen?

Reply:
Cleanup happens during the spin of the Constellation event loop here: https://github.com/mrobinson/servo/blob/7e11aff754449b4a888898aeef561ebac0356b41/components/constellation/constellation.rs#L782

Reply:
Oh. I see. You are right. I'll address this.